src/components/metadata/styles/metadata.scss

```diff
@@ -299,3 +300,8 @@ $list-inline-spacing: 0.2em;
   color: var(--color-button-text);
   font-size: 1.6em;
 }
+
+.react-json-view {
+  font-family: Consolas, Monaco, 'Courier New', Courier, monospace !important;
```

Comment:
Do we want to force `!important` here or should we prefix it with a more specific selector, e.g. `.pipeline-metadata__params-json-view`

Reply:
i am going to give this another go. I tried it before and it didn't override the font. so basically without important it applies the font of the react-json-view package. and i am overriding that class here. let me refine it on Monday again.

Reply:
i am going to give this another go. I tried it before and it didn't override the font. so basically without important it applies the font of the react-json-view package. and i am overriding that class here. let me refine it on Monday again.